onelogin_aws_cli/configuration.py

```diff
@@ -7,32 +7,49 @@
 class ConfigurationFile(configparser.ConfigParser):
     """Represents a configuration ini file on disk"""
 
+    DEFAULTS = dict(save_password=False)
+
     def __init__(self, config_file=None):
         super().__init__(default_section='defaults')
 
-        self['defaults'] = dict(save_password=False)
+        self[self.default_section] = dict(self.DEFAULTS)
 
         self.file = config_file
 
         if self.file is not None:
             self.load()
 
+    @property
+    def has_defaults(self) -> bool:
+        """True if the defaults section has settings beyond our defaults"""
```

Comment:
I... am confused...? what's the difference between the default section and the defaults?

Reply:
I can clarify the comment, but I just meant that if it's properly initialized, they should have something beyond `save_password` (i.e., "our defaults") which we inject into the defaults section.